### DISCUSSION TITLE
[bug] Pipe Operator should support byref type in this scenario

### DISCUSSION THREAD

**Discussion by Xyncgas** [fsharp/fslang-suggestions#1234] [General]
![image](https://user-images.githubusercontent.com/35312807/212532286-287ecd3c-4ba6-4965-83a6-aac5f20ec41e.png)
![image](https://user-images.githubusercontent.com/35312807/212532297-87812990-86c8-4ff9-9df4-defb44871442.png)
As you can see, I can't use the pipe operator and it made my code longer, the alternative way works it is however exactly the kind of things we would use a pipe operator for

**Comment by vzarytovskii** [comment]
This is not a bug, please create a suggestion for supporting this, or find existing one, so it can properly be discussed

**Comment by cartermp** [comment]
There's already and open language suggestion for this.
